### DISCUSSION TITLE
Cannot connect to Elasticache Redis

### DISCUSSION THREAD

**Discussion by anle20** [apache/airflow#34297] [Q&A]
Hi all,
We just upgraded airflow version from 2.5 to 2.7 and we met problem with connection from airflow to elasticache redis aws. We deployed airflow on ECS AWS

[ERROR/MainProcess] consumer: cannot connect to redis://xxx.cache.amazonaws.com:6379/0: Error 2 connecting to redis://xxx.cache.amazonaws.com:6379. No such file or directory.

I opened port on redis and I can use telnet to test it from worker instance to redis.

Do we need to add any environment vars for it?

**Comment by potiuk** [comment]
Hard to say what your configuration was but it looks like for some reason your past configuration for redis is not good for celery that is used in this version of Airflow or maybe somethign is wrong in your configuration and your redis cannot be contacted. Maybe there is a typo, or maybe networking configuration changed while you upgraded. Look at the Celery docs and their example configuration - compare it with celery section in Airflow and see if it looks right.
